CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # chapi-demo-wallet Changelog
 
+## 2.0.4 - 2023-08-16
```

Comment:
JFYI, this has to be set right before running the tools (the tool needs to see an uncommitted change in the changelog, where we put the release date), so we typically just leave a placeholder for the day until release time.

```suggestion
## 2.0.4 - 2023-08-dd
```